docs/release-notes/release-notes-v0.3.0.md

```diff
@@ -3,14 +3,15 @@
 **Release date: December 16, 2018**
 
 + Final release based on Lucene 6; upgrade to Lucene 7 planned next.
-+ Added runbooks for TREC 2018: Anserini and CENTRE groups.
++ TREC 2018 additions:
+  - Added runbooks for Anserini and h2oloo groups.
+  - Added code for News Track Background Linking Task.
```

Comment:
I roughly remember the index of Wash18 was also added in this version (by @Kytabyte and finalized by myself with some changes).

Reply:
Indeed: https://github.com/castorini/Anserini/blob/master/docs/regressions-log.md#november-30-2018
Will add